src/main.c

```diff
@@ -2408,6 +2412,23 @@ void handle_mouse_input() {
     }
 }
 
+
+float setJump(int megaJump, float dy)
+{
+    if ((megaJump) && (dy == 0))
+    {
+        dy = 40;
+        
+    }
+    else if (dy == 0)
+    {
+        dy = 8;
+        
+    }
+    
+    return dy;
+}
+
```

Comment:
proper formatting of if and else if statement is required just like it is followed on line 2254.